torch/cuda/nvtx.py

```diff
@@ -46,3 +47,26 @@ def mark(msg):
     if _libnvToolsExt() is None:
         raise RuntimeError('Unable to load nvToolsExt library')
     return lib.nvtxMarkA(ctypes.c_char_p(msg.encode("ascii")))
+
+class nvtxEventAttributes_t(ctypes.Structure):
+    """
+    A C struct containing essential attributes and optional
+    attributes about a CUDA event. 
+    """
+    _fields_ = [('version', ctypes.c_ushort),
+                ('size', ctypes.c_ushort),
+                ('colorType', ctypes.c_int),
+                ('color', ctypes.c_uint),
+                ('msgType', ctypes.c_int),
+                ('msg', ctypes.c_char_p)
+               ]
+
+    def __init__(self, message, color, version=DEFAULT,
+                 size, colorType=0xFF00FF00):
```

Comment:
You cannot have a positional argument after a keyword argument in python functions.